build_tools/buildkite/cmake/android/arm64-v8a/benchmark2.yml

```diff
@@ -131,13 +131,32 @@ steps:
   - label: "Comment benchmark results on pull request"
     commands: |
       git clean -fdx
+      # Fetch base branch for comparison.
+      git fetch -v --prune --no-recurse-submodules -- origin \
+        "$${BUILDKITE_PULL_REQUEST_BASE_BRANCH}"
+      git merge-base \
+        "origin/$${BUILDKITE_PULL_REQUEST_BASE_BRANCH}" \
+        "$${BUILDKITE_COMMIT}" > "./pr-base-commit"
       buildkite-agent artifact download "benchmark-results-*.json" ./
       buildkite-agent artifact download "compile-stats-results-*.json" ./
-      python3 build_tools/benchmarks/post_benchmarks_as_pr_comment.py \
+      # The script will handle the wildcard patterns.
+      python3 build_tools/benchmarks/generate_benchmark_comment.py \
         --verbose \
-        --query-base \
-        --benchmark_files benchmark-results-*.json \
-        --compile_stats_files compile-stats-results-*.json
+        --benchmark_files="benchmark-results-*.json" \
+        --compile_stats_files="compile-stats-results-*.json" \
+        --pr_number="$${BUILDKITE_PULL_REQUEST}" \
+        --pr_commit="$${BUILDKITE_COMMIT}" \
+        --pr_base_commit="$(cat ./pr-base-commit)" \
+        --build_url="$${BUILDKITE_BUILD_URL}" \
+        --comment_title="Abbreviated Android Benchmark Summary" \
+        --comment_type_id="bf8cdf94-a992-466d-b11c-778cbd805a22" \
```

Comment:
Where does this value come from?

Reply:
It's a random-generated id. I added the comment above to explain it.

If you think a readable id like `abbreviated-android-benchmark-summary` is better, I can switch to that. The downside is we might change the title in the future and make the id irrelevant.